src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -73,7 +73,13 @@ class WorkspaceMembersPage extends React.Component {
     }
 
     componentDidMount() {
-        const clientMemberEmails = _.keys(this.props.policyMemberList);
```

Comment:
The code from `componentDidMount` and `componentDidUpdate` is the same, we could extract it into a separate method, something like
```js
    /**
     * Get members for the current workspace
     */
    getWorkspaceMembers() {
        /**
         * clientMemberEmails should be filtered to only pass valid members, failure to do so
         * will remove all non-existing members that should be displayed (e.g. non-existing members that should display an error).
         * This is due to how calling `Onyx::merge` on array fields overwrites the array.
         * see https://github.com/Expensify/App/issues/12265#issuecomment-1307889721 for more context
         */
        const clientMemberEmails = _.keys(_.pick(this.props.policyMemberList, member => member.role));
        Policy.openWorkspaceMembersPage(this.props.route.params.policyID, clientMemberEmails);
    }
```